### PR TITLE
[#497] Modify overview i18n label and button styling

### PR DESCRIPTION
**Pull request by tangrammer**
Changed style 
Screenshot 2021-06-11 at 11 54 51

OnMouseEnter style
Screenshot 2021-06-11 at 12 01 38

### CODE REVIEW

File: sites/akvo-flow-web/src/util/Languages.js [hunk 1]
```diff
@@ -742,6 +742,10 @@ export const Locale = {
         en: "Confirm",
         de: "Bestätigen"
     },
+    overviews: {
+        en: "Overview and Download",
+        de: "Überblick und Download"
```
Comment (iperdomo) [inline sites/akvo-flow-web/src/util/Languages.js:747]: Does *Download* needs to be translated?

Comment (iperdomo) [reply sites/akvo-flow-web/src/util/Languages.js:747]: e.g. *Herunterladen*

Comment (tangrammer) [reply sites/akvo-flow-web/src/util/Languages.js:747]: Partner has specified this translation 🤔 as you can see in the related issue https://github.com/akvo/akvo-tech-consultancy/issues/497

Comment (iperdomo) [reply sites/akvo-flow-web/src/util/Languages.js:747]: Then, I have said nothing 🤐

**Pull request merged**